protocols/raft/src/test/java/io/atomix/protocols/raft/RaftTest.java

```diff
@@ -1372,6 +1487,13 @@ private TestPrimitive createPrimitive(RaftClient client, ReadConsistency consist
     return new TestPrimitiveImpl(proxy, registry);
   }
 
+  private CompletableFuture<Void> reduceFutures(Stream<CompletableFuture> stream) {
```

Comment:
couldn't we use something like this?

```
    public static <F extends CompletableFuture<?>> CompletableFuture<Void> allOf(Stream<F> cfs) {
        return CompletableFuture.allOf(cfs.toArray(CompletableFuture[]::new));
    }
```

Reply:
I changed it to use instead https://github.com/atomix/atomix/blob/dbad13756abfd997386c2d046a3d8bb8d45570d3/utils/src/main/java/io/atomix/utils/concurrent/Futures.java#L175